### PR TITLE
Change query to improve performance

### PR DESCRIPTION
**Pull request by crisely09**
This is a small change that seems to run in a few seconds, just removing the backward properties.

### CODE REVIEW

**Comment by crisely09** [comment]
One doubt I have is if the annotations should now belong to `mainResources` or stay as `otherResources`?

File: tests/docker/config/resource-types.json [hunk 1]
```diff
@@ -36,7 +36,8 @@
   "https://neuroshapes.org/NeuronCount",
   "https://neuroshapes.org/NeuronDensity",
   "https://neuroshapes.org/NeuronMorphology",
-  "https://neuroshapes.org/NISSLImageDataLayer",
+  "https://bbp.epfl.ch/ontologies/core/bmo/NeuronMorphologyFeatureAnnotation",
```
Comment (imsdu) [inline tests/docker/config/resource-types.json:39]: You don't need to add a new type if they are not indexed separately from the `NeuronMorphology`

File: tests/docker/config/resource-types.json [hunk 2]
```diff
@@ -36,7 +36,8 @@
...
+  "https://t neuroshapes.org/NISSLImageDataLayer",
```
[the remark this answers is not in the document]
Comment (crisely09) [reply tests/docker/config/resource-types.json]: fixed

**Comment by crisely09** [comment]
each morphology takes ~ 3s for the query, here the important change is the fix for `Total Width`, so it can be used in the front-end

**Comment by imsdu** [comment]
Closed in favour of #4687

**Pull request closed without merging**